src/store.jsx

```diff
@@ -0,0 +1,14 @@
+import { createStore, applyMiddleware } from 'redux';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved
+import { composeWithDevTools } from 'redux-devtools-extension';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers';
+
+const middleware = [thunk];
+
+const store = createStore(
```

Comment:
If you are going to use react-redux I highly recommend use redux-toolkit instead of the normal redux
[https://redux-toolkit.js.org/](url)